firmware/controllers/algo/launch_control.cpp

```diff
@@ -55,7 +55,7 @@ bool LaunchControlBase::isInsideSwitchCondition() const {
  * then we have to return true, and trust that we would disable by other condition!
  */ 
 bool LaunchControlBase::isInsideSpeedCondition() const {
-	int speed = getVehicleSpeed(PASS_ENGINE_PARAMETER_SIGNATURE);
+	int speed = static_cast<int>(Sensor::get(SensorType::VehicleSpeed).value_or(0));
```

Comment:
no explicit cast required, perfectly safe to just assign a float to an int

Reply:
on second thought, why is `speed` an int at all? Just make it a float.

Reply:
im will remove cast and make speed type changing with another PR and commit. Ok?